site/content/docs/4.3/migration.md

```diff
@@ -45,6 +45,8 @@ Changes to our source Sass files and compiled CSS.
 - `color-yiq()` function is renamed to `color-contrast()` since it's not related to YIQ colorspace anymore — [See #30168](https://github.com/twbs/bootstrap/pull/30168/) — and related variables are renamed alongside:
   - `$yiq-contrasted-threshold` is renamed `$min-contrast-ratio`,
   - `$yiq-text-dark` and `$yiq-text-light` are respectively renamed `$color-contrast-dark` and `$color-contrast-light`.
+- Linear gradients are simplified when gradients are enabled and therefor, `gradient-bg()` now only accepts an optional `$color` parameter.
+- `bg-gradient-variant()` mixin is removed since the `.bg-gradient` class can now be added to add gradients to elements instead of the `.bg-gradient-*` classes.
```

Comment:
```suggestion
- `bg-gradient-variant()` mixin is removed since the `.bg-gradient` class can now be used to add gradients to elements instead of the `.bg-gradient-*` classes.
```